README.md

```diff
@@ -661,39 +644,7 @@ As expected, no entity decrypts the `data`. The `raw_data` is stored as plaintex
 
 So, the data is stored in an encrypted form, but it is transparent for the Python application.
 
-### 2.6 Encrypt the data without Zones
```

Comment:
we show how user can read/write using one public key and acra's client, and don't show how it looks if it goes to acra configured to process another client's data... Let's describe here what would be get with another TLS certificate, from another client. It's the same as another zone but in client_id terminology